src/react/boosts/BoostTextComponents.tsx

```diff
@@ -78,3 +79,89 @@ export const PaymentFAQText: React.FunctionComponent = props => (
     <a href="#TODO">What information can others see about my payment?</a>
   </>
 );
+
+export const PaymentConfirmTransactionText: React.FunctionComponent = props => (
+  <>
+    <h3>Let's Boost</h3>
+    <p>Your Wallet has popped up a new window. Confirm the transaction in the Wallet to complete the Boost.</p>
+  </>
+);
+
+export interface BoostPaymentTextProps {
+  newsroomName?: string;
+  etherToSpend?: number;
+  usdToSpend?: number;
+  boostURL?: string;
+  hideModal?(): void;
+}
+
+export const PaymentSuccessText: React.FunctionComponent = props => <ModalHeading>Payment Successful!</ModalHeading>;
+
+export const PaymentErrorText: React.FunctionComponent = props => (
+  <>
+    <ModalHeading>Payment Failed</ModalHeading>
+    <ModalContent>Your transaction failed. Please try again.</ModalContent>
+  </>
+);
+
+export const PaymentEthConfirmationText: React.FunctionComponent<BoostPaymentTextProps> = props => (
+  <>
+    <p>
+      Thank you! {props.newsroomName} has received your Boost of {props.etherToSpend} ETH ({props.usdToSpend}} USD)
+    </p>
+  </>
+);
+
+export const PaymentShareText: React.FunctionComponent<BoostPaymentTextProps> = props => (
+  <>
+    <p>
+      Tell your friends about your Boost! <a href={props.boostURL}>Share a link to the Boost</a>
+    </p>
+  </>
+);
+
+export const PaymentJoinCivilText: React.FunctionComponent = props => (
+  <p>
+    Want to create a Civil account and keep track of your boost? Join the Civil Community and take part in helping to
```

Comment:
hhmm... we don't actually keep track of boosts by account.. might need to rephrase this until we do

Reply:
This can be deleted it was part of an old design it's not actually being used